test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -56,6 +57,7 @@ var _ = SIGDescribe("Upgrade [Feature:Upgrade]", func() {
 	// in a "Describe".
 	ginkgo.Describe("master upgrade", func() {
 		ginkgo.It("should maintain a functioning cluster [Feature:MasterUpgrade]", func() {
+			e2epv.SkipIfNoDefaultStorageClass(f.ClientSet)
```

Comment:
Do these tests use storageclass? I think only the statefulset_upgrade and storage_upgrade test do.

Reply:
Yeah if you see line 48 it runs PV tests

Reply:
hm it's a bit unfortunate that we have to disable all upgrade tests though. cc @jpbetz do we have a plan overall to disable upgrade jobs entirely from k/k?